src/FakeItEasy/Core/FakeManager.cs

```diff
@@ -93,7 +94,7 @@ public virtual IEnumerable<ICompletedFakeObjectCall> RecordedCallsInScope
             get { return FakeScope.Current.GetCallsWithinScope(this); }
         }
 
-        internal List<ICompletedFakeObjectCall> AllRecordedCalls
+        internal IEnumerable<ICompletedFakeObjectCall> AllRecordedCalls
         {
             get { return this.recordedCallsField; }

```

Comment:
:+1: for renaming the field.

To make sure I understand, can you explain why you prefer a method call to a property for accessing the recorded calls.

(And for the `ToList`, you're just worried that callers will cast the enumerable back to a queue and mutate? I think we've not been protecting internal members in this way so far. I'm not against doing so, wouldn't've worried about the internal clients being so ill-behaved.)